cogs/moderation.py

```diff
@@ -293,6 +298,96 @@ async def clear(self, ctx: Context, amount: int) -> None:
         await asyncio.sleep(2.5)
         await message.delete()
 
+    @command()
+    @has_permissions(manage_messages=True)
+    async def shift(self, ctx, count: int, target: TextChannel, copy: bool = False) -> None:
+        """Copy or Move specified messages amount to specified channel"""
+        messages = []
+        async for message in ctx.message.channel.history(limit=count):
```

Comment:
```suggestion
        async for message in ctx.channel.history(limit=count):
```
I don't think the message is necessary here.